conf/crick.config

```diff
@@ -0,0 +1,23 @@
+/*

```

Comment:
Could you maybe contribute this profile for consistency to our main template as well? Would give you access to all other pipelines once syncing is tested and established very soon?  Similar to the PR I made a couple of days ago:

https://github.com/nf-core/tools/pull/201#issuecomment-440580568

Reply:
This will now be sorted with the addition of `nf-core/configs`